Samples/PublicSamples/RecordingBot/docs/setup/policy.md

```diff
@@ -5,109 +5,121 @@ The following steps should be done by an Office 365 administrator.
+   -ApplicationId <the Application Id of the Bot Application Registration>
+```
+The Application Id might not be from your own Microsoft Entra ID,
+but from the Microsoft Entra ID that hosts this Bot.
+So basically from the Service Provider.
+
+You will now have to Synchronize this Application Instance into the Agent Provisioning Service
+```powershell
+Sync-CsOnlineApplicationInstance
+    -ObjectId <the object id of the Application Instance>
+    -ApplicationId <the application id of the remote application registration>
+```
+You can get the Object Id by executing the `Get-CsOnlineApplicationInstance -Displayname <the name you provided>` and checking the output.
+The Object Id will also be displayed after the `New-CsOnlineApplicationInstance` command.
+The Application Id is the same Application Id you used to create the Application Instance.
+
+### Create the policy
+
+[New-CsTeamsComplianceRecordingPolicy](https://learn.microsoft.com/en-us/powershell/module/skype/new-csteamscompliancerecordingpolicy?view=skype-ps)
+```powershell
+New-CsTeamsComplianceRecordingPolicy
+   -Identity <provide a name for your policy>
+   -Enabled $true
+```
+With this, you will just have a policy.
+You can already assign this policy to users, but it will not do anything,
+because it does not have any Recording Applications assigned to it yet.
+
+### Create the Recording Application
+
+[New-CsTeamsComplianceRecordingApplication](https://learn.microsoft.com/en-us/powershell/module/skype/new-csteamscompliancerecordingapplication?view=skype-ps)
+```powershell
+New-CsTeamsComplianceRecordingApplication
+   -Parent <Recording Policy Name>
+   -Id <Application Instance Object Id>
+```
+The <Recording Policy Name> is the Parameter `Identity` from the `New-CsTeamsComlianceRecordingPolicy`.
+So the Parent of a Recording Application is a Recording policy.
+The <Application Instance Object Id> is the Object ID of the Object created with `New-CsOnlineApplicationInstance`.
+So basically you now have
+- assigned an Application in your own Entra (Application Instance)
+ - wich points to an application on an external Entra (Application Id that was assigned to Application Instance)
+- to a Teams Compliance Policy (by using its name)
+
+### Activate the policy
+
+[Grant-CsTeamsComplianceRecordingPolicy](https://learn.microsoft.com/en-us/powershell/module/skype/grant-csteamscompliancerecordingpolicy?view=skype-ps)
+```powershell
+Grant-CsTeamsComplianceRecordingPolicy
+     -Identity <User Principal name of the Application Instance>
+     -PolicyName <Name of the Recording Policy>
+```
+The Identity is the same as passed into the `UserPrincipalName` parameter of the `New-CsOnlineApplicationInstance` command.
+The PolicyName is the same as passed into the `Identity` parameter of the `New-CsTeamsComplianceRecordingPolicy` command.
+
+## Use the policy
+
+To be able to use the Policy, you will need to assign Users or Groups to this policy.
```

Comment:
How can we assign the Policy to Users or Groups?

Reply:
Outside of scope of this documentation